app/src/org/commcare/views/HorizontalMediaView.java

```diff
@@ -27,137 +26,112 @@
  *
  * @author wspride
  */
-
 public class HorizontalMediaView extends RelativeLayout {
-    private static final String t = "AVTLayout";
+    private static final String TAG = HorizontalMediaView.class.getSimpleName();
 
-    private TextView mTextView;
-    private AudioButton mAudioButton;
-    private ImageView mImageView;
-    private final EvaluationContext ec;
     private final int iconDimension;
-
-    public static final int NAVIGATION_NONE = 0;
-    public static final int NAVIGATION_NEXT = 1;
-
+    private AudioButton audioButton;
 
     public HorizontalMediaView(Context c) {
-        this(c, null);
-    }
-
-    private HorizontalMediaView(Context c, EvaluationContext ec) {
         super(c);
-        mTextView = null;
-        mAudioButton = null;
-        mImageView = null;
-        this.ec = ec;
-        this.iconDimension = (int)getResources().getDimension(R.dimen.menu_icon_size);
+        this.iconDimension = (int) getResources().getDimension(R.dimen.menu_icon_size);
```

Comment:
Thought our official style policy on casting was no space, but can't remember where we actually have that stuff documented so not positive..